packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -287,14 +287,13 @@ export default class Roap extends StatelessWebexPlugin {
     // When reconnecting, it's important that the first roap message being sent out has empty media id.
     // Normally this is the roap offer, but when TURN discovery is enabled,
     // then this is the TURN discovery request message
-    const sendEmptyMediaId = reconnect && !meeting.config.experimental.enableTurnDiscovery;
```

Comment:
In the description, it is mentioned that the locus doesn't expect the media id during reconnection, which is causing the problem.
However, in the comment above, it's specified that the media id should NOT be sent. So it seems like this is expected behavior from the locus. This leads to the fact that this is a bug on our SDK.

Can we know why we had the check for enableTurnDiscovery and why we are removing that?

Reply:
this fix is wrong, like the comment in the code says we need to send empty mediaId with the first Roap message that we send on reconnection - tha is the OFFER when we are not using TURN TLS or the TURN_DISCOVEEY_MESSAGE if we are using TURN TLS. This existing code on master is not correct, because it only takes into account the config value for `enableTurnDiscovery` and doesn't take into account that TURN discovery is skipped if reachability succeeds. This was fixed on beta branch in this PR: https://github.com/webex/webex-js-sdk/pull/2782 It is possible that you're hitting the same issue in SPARK-441781 (the logs attached there show that TURN discovery was enabled but skipped), so you need to port that fix or just start using beta branch.

Reply:
Thanks @marcin-bazyl . @Shreyas281299 I suggest we cherry pick the fix from beta and apply it here.